tests/test_elasticsearch.py

```diff
@@ -70,3 +70,11 @@ def test_search_response_is_empty_if_indexer_fails(self):
                                  headers=self.headers)
             result = resp.json
             assert result == {}
+
+    def test_invalid_search_query(self):
+        body = {"whatever": {"wrong": "bad"}}
+        resp = self.app.post_json("/buckets/bid/collections/cid/search",
+                                  body,
+                                  headers=self.headers,
+                                  status=400)
+        assert resp.json["message"] == "body: Unknown key for a START_OBJECT in [whatever]."
```

Comment:
Nit: maybe we'd want a slightly more self-explanatory message (aka I don't have a clue what a START_OBJECT is)